core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -2663,6 +2663,14 @@ public void testPrepare()
                         table(QualifiedName.of("foo")))));
     }
 
+    @Test
+    public void testPrepareDropView()
+    {
+        assertStatement("PREPARE statement1 FROM DROP VIEW IF EXISTS \"catalog-test\".\"test\".\"foo\"",
```

Comment:
could you also add a test where the statement fails if not quoted?

Reply:
@lukasz-stec Thanks
add the test case in following PR.